Presentation.Web/App_Start/LogConfig.cs

```diff
@@ -13,6 +21,10 @@ public static class LogConfig
 
         private static ILogger ConfigureAndCreateSerilogLogger()
         {
+            //Uncomment to log the serilog errors in the Output window
```

Comment:
either make a config setting (you can test whether or not we are in production - that setting exists) for it or remove this green code. 

Green code has the problem, that it is not maintained along with the rest of the application, so if "what made it work" when it was created, suddenly changes, the day it is needed, then we have a hidden problem.

Better to have a flag/appSetting which enables it based on e.g. "running locally" or perhaps a specialized setting `enableSerilogSelfLog`